android/app/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
     }
 
     defaultConfig {
-        applicationId "foundation.privacybydesign.irmamobile.alpha"
+        applicationId "org.irmacard.cardemu"
```

Comment:
Did you discuss this with @w-ensink? Because we have a alpha app also available in the Play Store environment if I am not mistaken.